libs/ngxtension/host-binding/src/host-binding.ts

```diff
@@ -60,7 +60,13 @@ export function hostBinding<T, S extends Signal<T> | WritableSignal<T>>(
 					);
 					break;
 				case 'attr':
-					renderer.setAttribute(element, property, String(value));
+					const v = value ?? null;
+
+					if (v === null) {
```

Comment:
nit: maybe we can just `== null` check for `value` to remove attribute if `value` is `undefined` or `null`?

Reply:
I'm not against that